services/app-api/utils/constants/constants.ts

```diff
@@ -55,3 +55,67 @@ export const bucketTopics = {
 };
 
 export const formTemplateTableName = process.env.FORM_TEMPLATE_TABLE_NAME!;
+
+// Form item ids we should not copy over.
+export const MCPARFieldIDBlocklist = {
+  matchString: [
+    "contactName",
+    "contactEmailAddress",
+    "submitterName",
+    "submitterEmailAddress",
+    "reportSubmissionDate",
+    "reportingPeriodStartDate",
+    "reportingPeriodEndDate",
+    "state_statewideMedicaidEnrollment",
+    "state_statewideMedicaidManagedCareEnrollment",
+    "state_focusedProgramIntegrityActivitiesConducted",
+    "state_excludedEntityIdentifiedInFederalDatabaseCheck",
+    "state_excludedEntityIdentificationInstancesSummary",
+    "state_submittedDataAuditResults",
+    "program_contractDate",
+    "program_contractUrl",
+    "program_enrollment",
+    "program_enrollmentBenefitChanges",
+    "program_encounterDataCollectionValidationBarriers",
+    "program_networkAdequacyChallenges",
+    "program_networkAdequacyGapResponseEfforts",
+    "program_prohibitedAffiliationDisclosure",
+    "plan_enrollment",
+    "plan_medicaidEnrollmentSharePercentage",
+    "plan_medicaidManagedCareEnrollmentSharePercentage",
+    "plan_medicalLossRatioPercentage",
+    "plan_medicalLossRatioReportingPeriod",
+    "plan_ltssUserFiledCriticalIncidentsWhenPreviouslyFiledAppeal",
+    "plan_encounterDataSubmissionTimelinessCompliancePercentage",
+    "plan_encounterDataSubmissionHipaaCompliancePercentage",
+    "plan_dedicatedProgramIntegrityStaff",
+    "plan_stateFairHearingRequestsFiled",
+    "plan_stateFairHearingRequestsWithFavorableDecision",
+    "plan_stateFairHearingRequestsWithAdverseDecision",
+    "plan_stateFairHearingRequestsRetracted",
+    "plan_stateFairHearingRequestsWithExternalMedicalReviewWithFavorableDecision",
+    "plan_stateFairHearingRequestsWithExternalMedicalReviewWithAdverseDecision",
+    "plan_ltssUserFiledCriticalIncidentsWhenPreviouslyFiledGrievance",
+    "plan_mfcuProgramIntegrityReferrals",
+    "plan_openedProgramIntegrityInvestigations",
+    "plan_programIntegrityInvestigationsToEnrolleesRatio",
+    "plan_resolvedProgramIntegrityInvestigations",
+    "plan_resolvedProgramIntegrityInvestigationsToEnrolleesRatio",
+    "plan_programIntegrityReferralsPerThousandBeneficiaries",
+    "plan_overpaymentRecoveryReportDescription",
+    "sanction_interventionType",
+    "sanction_interventionTopic",
+    "sanction_planName",
+    "sanction_interventionReason",
+    "sanction_noncomplianceInstances",
+    "sanction_dollarAmount",
+    "sanction_assessmentDate",
+    "sanction_remediationCompleted",
+    "sanction_remediationDate",
+    "sanction_correctiveActionPlan",
+    "qualityMeasure_reportingPeriod",
+    "qualityMeasure_reportingPeriodStartDate",
+    "qualityMeasure_reportingPeriodEndDate",
+  ],
+  wildcard: ["appeals", "grievances", "sanctions"],
```

Comment:
Question: I don't understand the purpose of the wildcard section here. Do we have auto-generated field IDs that can't be predicted? And if so, could the matching logic be made any more specific (For example, `key.startsWith(wildCard)` vs `key.includes(wildCard)`)? As it is, I'm a little bit worried about false positives.

Reply:
basically, it's cutting down on listing out each id in the blocklist. According to the document, it sounds like the stakeholders will never want to copy over anything that regards sanctions, appeals or grievances due to the nature of the data. so we're not super concerned about false positives.
Is that cool? Or do you think it's still worth it to update the blocklist with each ID to ignore

Reply:
For me, that raises more questions. With the current code:
1. We would exclude 11 of the 12 fields in the Appeals Overview drawerForm - but we would include `plan_ltssUserFiledCriticalIncidentsWhenPreviouslyFiledAppeal`, as it doesn't include `appeals` exactly.
   * Although this is remediated by including that field in the matchString array
2. Similarly, we would exclude 4 of the 5 fields in the Grievances Overview drawerForm - but we would include `plan_ltssUserFiledCriticalIncidentsWhenPreviouslyFiledGrievance`, as it doesn't include `grievances` exactly.
   * Although this is also remediated by including that field in the matchString array
3. We include `program_standardAppealTimelyResolutionDefinition` and `program_expeditedAppealTimelyResolutionDefinition` - would those be intentional?
4. We include `program_grievanceTimelyResolutionDefinition` - probably not intentional?
5. The "sanctions" wildcard does nothing, as it doesn't appear in any field IDs
6. We include `sanction_interventionType-othertext` and `sanction_interventionTopic-othertext`
   * Although this could be fixed by changing the `sanctions` wildcard to `sanction` only
   * And if we did that, we may as well remove the sanction-specific IDs from the matchString array

I'm wondering if it would make more sense to include/exclude fields based on which page or form contains them - that would be a significant code change at this point, as it would require considering the form template rather than just the IDs. But with the current implementation, even if we patched up all these little edge cases, it really lends itself to creating more in the future. As the report changes over time, I anticipate bug reports like "this whole page copied from the previous year, except this one question, why?" and "this page wasn't supposed to copy from the previous year, except this one nested question did, why?"